instance_skel_types.d.ts

```diff
@@ -5,6 +5,51 @@ export interface CompanionSystem extends EventEmitter {}
 
 export type InputValue = number | string | boolean
 
+export type CompanionBank = CompanionBankPage | CompanionBankPNG | CompanionBankPreset
+
+export interface CompanionBankPage {
+	style: 'pageup' | 'pagedown' | 'pagenum'
+}
+
+export interface CompanionBankPNG {
+	style: 'png'
+	text: string
+	size: 'auto' | '7' | '14' | '18' | '24' | '30' | '44'
+	alignment:
+		| 'left:top'
+		| 'center:top'
+		| 'right:top'
+		| 'left:center'
+		| 'center:center'
+		| 'right:center'
+		| 'left:bottom'
+		| 'center:bottom'
+		| 'right:bottom'
+	pngalignment:
+		| 'left:top'
+		| 'center:top'
+		| 'right:top'
+		| 'left:center'
+		| 'center:center'
+		| 'right:center'
+		| 'left:bottom'
+		| 'center:bottom'
+		| 'right:bottom'
+	png64?: string
+	color: number
+	bgcolor: number
+	latch: boolean
+	relative_delay: boolean
+}
+
+export interface CompanionBankPreset {
```

Comment:
I'm not keen on CompanionBankPreset and CompanionBankPNG being separate types, as they are the same thing, but as the preset version wants a lot of the properties to be optional then it becomes a little messy.

I don't expect you to do anything about this right now, I may have a play myself later